packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -173,7 +173,7 @@ const getConfigFromEnv = () => {
     allowLocalUrls: process.env.COMPANION_ALLOW_LOCAL_URLS === 'true',
     // cookieDomain is kind of a hack to support distributed systems. This should be improved but we never got so far.
     cookieDomain: process.env.COMPANION_COOKIE_DOMAIN,
-    streamingUpload: process.env.COMPANION_STREAMING_UPLOAD === 'true',
+    streamingUpload: process.env.COMPANION_STREAMING_UPLOAD ? process.env.COMPANION_STREAMING_UPLOAD === 'true' : undefined,
```

Comment:
because we want it to default to true if the env variable is not set ( but it should still be possible to). if we return false here it will override the default which is true afaik

config gets merged here: https://github.com/transloadit/uppy/blob/f75d3f1ef5d1b0216e365ea8e5a924db872b059d/packages/%40uppy/companion/src/standalone/helper.js#L233